packages/sync-utils/src/sanity/syncSanityDocument.ts

```diff
@@ -115,7 +116,8 @@ const mergeExistingFields = (
 
 export const createSyncSanityDocument = (
   client: SanityClient,
-  cache: SanityCache
+  cache: SanityCache,
+  shopifyClient: ShopifyClient
```

Comment:
I'd like to avoid using any Shopify client things within `/src/sanity`... There is no issue with what you're doing here - but I want to discourage others from doing any shopify fetching from within these functions.

How about we create a `SaneShopifyContext` interface that (for now) is just:

```
interface SaneShopifyContext {
  shopName: string
}
```

and pass that in instead?

Reply:
Ok let me know if it's better not